src/binding.cc

```diff
@@ -366,24 +366,38 @@ v8::Promise* v8__Promise__Then2(v8::Promise* self,
   return maybe_local_to_ptr(self->Then(context, on_fulfilled, on_rejected));
 }
 
-v8::PromiseRejectEvent 
-v8__PromiseRejectMessage__GetEvent(const v8::PromiseRejectMessage &self)
-{
+v8::PromiseRejectEvent v8__PromiseRejectMessage__GetEvent(
+    const v8::PromiseRejectMessage& self) {
   return self.GetEvent();
 }
 
-v8::Promise*
-v8__PromiseRejectMessage__GetPromise(const v8::PromiseRejectMessage &self)
-{
+v8::Promise* v8__PromiseRejectMessage__GetPromise(
+    const v8::PromiseRejectMessage& self) {
   return local_to_ptr(self.GetPromise());
 }
 
-v8::Value*
-v8__PromiseRejectMessage__GetValue(const v8::PromiseRejectMessage &self)
-{
+v8::Value* v8__PromiseRejectMessage__GetValue(
+    const v8::PromiseRejectMessage& self) {
   return local_to_ptr(self.GetValue());
 }
 
+v8::Isolate* v8__PropertyCallbackInfo__GetIsolate(
+    const v8::PropertyCallbackInfo<v8::Value>* self) {
+  return self->GetIsolate();
+}
+
+v8::Object* v8__PropertyCallbackInfo__This(
+    const v8::PropertyCallbackInfo<v8::Value>* self) {
+  return local_to_ptr(self->This());
+}
+
+v8::ReturnValue<v8::Value>* v8__PropertyCallbackInfo__GetReturnValue(
+    const v8::PropertyCallbackInfo<v8::Value>* self) {
+  v8::ReturnValue<v8::Value>* rv =
+      new v8::ReturnValue<v8::Value>(self->GetReturnValue());
```

Comment:
Actually does not look good after all (memory leak)

Reply:
Yikes, we already landed the same leak in #67

Reply:
What’s the leak? We need a LSAN check...